contribs/gmf/src/contextualdata/component.js

```diff
@@ -196,7 +199,7 @@ exports.Controller_.prototype.setContent_ = function(coordinate) {
   const getRasterError = function(resp) {
     console.error('Error on getting the raster.');
   };
-  this.gmfRaster_.getRaster(coordinate).then(
+  this.gmfRaster_.getRaster(coordinate, this.gmfContextualdataOptions.rasterParams || {}).then(
```

Comment:
```suggestion
  this.gmfRaster_.getRaster(coordinate, this.gmfContextualdataOptions.rasterParams).then(
```

The `getRaster` function can already accept `undefined` as param